lib/web/terminal.go

```diff
@@ -361,6 +361,13 @@ func (t *TerminalHandler) promptMFAChallenge(ws *websocket.Conn) client.PromptMF
 				})
 			}
 			chal = &auth.MFAAuthenticateChallenge{
+				AuthenticateChallenge: &u2f.AuthenticateChallenge{
+					// Get the common challenge fields from the first item.
+					// All of these fields should be identical for all u2fChals.
+					Challenge: u2fChals[0].Challenge,
+					AppID:     u2fChals[0].AppID,
```

Comment:
Will this indexing always pass? Can you ever get empty `u2fChals`?

Reply:
yes it will always pass b/c we do a length check before we enter this switch case: https://github.com/gravitational/teleport/blob/master/lib/web/terminal.go#L352